build_tools/benchmarks/common/benchmark_config_test.py

```diff
@@ -18,12 +18,16 @@ class BenchmarkConfigTest(unittest.TestCase):
 
   def setUp(self):
     self.build_dir = tempfile.TemporaryDirectory()
+    self.build_dir_name = os.path.realpath(self.build_dir.name)
     self.tmp_dir = tempfile.TemporaryDirectory()
-    self.normal_tool_dir = os.path.join(self.build_dir.name, "normal_tool")
+    self.tmp_dir_name = os.path.realpath(self.tmp_dir.name)
+    self.normal_tool_dir = os.path.join(self.build_dir_name, "normal_tool")
     os.mkdir(self.normal_tool_dir)
-    self.traced_tool_dir = os.path.join(self.build_dir.name, "traced_tool")
+    self.traced_tool_dir = os.path.join(self.build_dir_name, "traced_tool")
```

Comment:
We could consider using pathlib here. That may be more portable

Reply:
Not 100% but we may need to rewrite BenchmarkConfig? It is possible converting them to `pathlib.Path` will work fine but I wanted to avoid making singificant changes.

Reply:
Yeah for sure. Not suggesting it necessarily needs to happen in this PR. @pzread could you take a look at this? It may be better to drop the realpath calls from BenchmarkConfig itself, but for now this LGTM